GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -600,12 +592,6 @@ protected virtual void Dispose(bool disposing)
                     this.indexParsingThread.Dispose();
                     this.indexParsingThread = null;
                 }
-

```

Comment:
Are we no longer responsible for disposing?

Reply:
Nope. This class now takes the interface in the constructor and that interface isn't disposable and the class `PlaceholderTable` isn't disposable either since it just uses the connections that the `GVFSDatabase` manages.  The `GVFSDatabase` class is the only one that needs to be disposed in `InProcessMount.UnmountAndStopWorkingDirectoryCallbacks`